src/Jobs/CheckForUpdatesJob.php

```diff
@@ -52,4 +54,19 @@ public function process()
         // mark job as completed
         $this->isComplete = true;
     }
+
+    /**
+     * @inheritdoc
+     */
+    public function afterComplete()
+    {
+        // Queue a new job to run in the future
+        $injector = Injector::inst();
+        $queuedJobService = $injector->get(QueuedJobService::class);
+        $startAfter = new DateTime('+24 hours');
+        $queuedJobService->queueJob(
+            $injector->create(CheckForUpdatesJob::class),
+            $startAfter->format(DateTime::ISO8601)
```

Comment:
Any way you can use `DBDatetime` for this?

Reply:
I will use DBDatetime to get "now" and add 24 hours. That will mean "mock now" is considered.